### PR TITLE
[Edit] update to brightness.md added image example

### LINKED ISSUE #3744: [Edit] CSS [Edit] CSS Filter Functions: brightness()
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to a term within CSS Filter Functions. The term entry lives in docs/content/css/concepts/filter-functions/terms/brightness/brightness.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Atharva1723**
Resolves #3744

### Description

Added new png image to show the example

image

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [✔ ] All writings are my own.
- [✔ ] My entry follows the Codecademy Docs style guide.
- [✔ ] My changes generate no new warnings.
- [✔ ] I have performed a self-review of my own writing and code.
- [✔ ] I have checked my entry and corrected any misspellings.
- [✔ ] I have made corresponding changes to the documentation if needed.
- [✔ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✔ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✔ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Atharva1723 The brightness filter example that you added looks correct. But no need for any of the remaining textual content. To distinguish between the images you can add a sentence within the entry that states which image has the brightness filter.

Also, please make sure that the brightness percentage you used is reflected in the example code snippet.

**Comment by Atharva1723** [comment]
@ishg-153 I had reffered to the existing blur filter markdown

**Comment by Atharva1723** [comment]
Will make the requested changes

File: content/css/concepts/filter-functions/terms/brightness/brightness.md
Comment (ishg-153) [inline content/css/concepts/filter-functions/terms/brightness/brightness.md:40]:
```suggestion
![CSS brightness filter example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunctions-brightness-example.png)

Here, the image to the left is the original image and the one to the right is the same image with `brightness(125%)`. 
```

Adding something like this will clearly differentiate between the images.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Atharva1723 You hadn't addressed all of the points from the previous review. I have added the suggestions again. Please read them carefully and make all the changes.

**Comment by ishg-153** [comment]
@Atharva1723 Can you confirm the brightness percentage you used in your code that rendered the output you've added to here.

**Comment by Atharva1723** [comment]
> @Atharva1723 Can you confirm the brightness percentage you used in your code that rendered the output you've added here?

Yes I have added the image with brightness 100%

**Comment by Atharva1723** [comment]
@SSwiniarski @KTom101 Please review this PR Thank You :)

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Atharva1723 In the `css-filterfunctions-brightness-example.png` image you've added, there is text content that renders as "codecademy" in the output right above the Codecademy logo image. You still need to remove that.

**Comment by Atharva1723** [comment]
> @Atharva1723 In the `css-filterfunctions-brightness-example.png` image you've added, there is text content that renders as "codecademy" in the output right above the Codecademy logo image. You still need to remove that.

I think that should be only there because I had applied your suggestions and that's giving lint errors

**Comment by ishg-153** [comment]
> @SSwiniarski @KTom101 Please review this PR Thank You :)
Hi @Atharva1723. Once I approve your changes from the first review someone will perform a second review. Please try and be patient.

**Comment by Atharva1723** [comment]
@ishg-153 you can also see my previous commits the suggestions you had given me was applied by me

**Comment by ishg-153** [comment]
> I think that should be only there because I had applied your suggestions and that's giving lint errors

What kind of error are you encountering?

**Comment by Atharva1723** [comment]
Wait I will apply your suggestions

**Comment by ishg-153** [comment]
Hey @Atharva1723. I have an output image as per the requirements. Can you give me push access to your repo so that I can make the change and send this for a second review?

**Review by ishg-153 — APPROVED** [review]

**Review by ishg-153 — COMMENTED** [review]
I made the leftover changes to reduce confusion. This is good for a second review now.

**Review by caupolicandiaz — APPROVED** [review]
@Atharva1723 @ishg-153 LGTM

**Pull request merged**
